py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/convert_gtfs_rt.py

```diff
@@ -65,13 +62,19 @@ def convert(self, files: list[str]) -> list[tuple[str, pyarrow.Table]]:
             self.detail.empty_table(), schema=self.detail.export_schema
         )
 
+        # this is the default pool size for a ThreadPoolExecutor as of py3.8
+        cpu_count = cast(
+            int, os.cpu_count() if os.cpu_count() is not None else 1
+        )
+        pool_size = min(32, cpu_count + 4)
```

Comment:
decided to update this to match the py spec. should give us a modest speed boost.

i used this value in the s3 move stuff too.